completions/hub.completion.sh

```diff
@@ -2,14 +2,14 @@
 # This script complements the completion script that ships with git.
 
 # If there is no git tab completion, but we have the _completion loader try to load it
-if ! declare -F _git > /dev/null && declare -F _completion_loader > /dev/null; then
+if ! _omb_util_function_exists _git > /dev/null && _omb_util_function_exists _completion_loader > /dev/null; then
   _completion_loader git
 fi
 
 # Check that git tab completion is available
-if declare -F _git > /dev/null; then
+if _omb_util_function_exists _git > /dev/null; then
   # Duplicate and rename the 'list_all_commands' function
-  eval "$(declare -f __git_list_all_commands | \
+  eval "$(_omb_util_function_exists __git_list_all_commands | \
```

Comment:
We cannot replace this because we want the output of `declare -f` here. Actually, we can only safely replace the combination of `declare -F CMD >/dev/null`, `declare -F CMD &>/dev/null`, etc. including redirections to `_omb_util_function_exists CMD`.